plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -17,12 +17,11 @@
 package com.google.cloud.tools.eclipse.appengine.newproject;
 
 import com.google.cloud.tools.eclipse.appengine.facets.AppEngineStandardFacet;
-import com.google.cloud.tools.eclipse.appengine.libraries.LibraryClasspathContainerResolverJob;
-import com.google.cloud.tools.eclipse.appengine.libraries.model.Library;
+
```

Comment:
Until the public google stylesheet from https://github.com/google/styleguide catches up with this rule, I wouldn't worry too much about import order.